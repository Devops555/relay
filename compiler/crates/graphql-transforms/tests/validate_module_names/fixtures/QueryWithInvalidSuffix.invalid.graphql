#expected-to-throw
query QueryWithInvalidSuffixMutation {
    me {
        name
    }
}
