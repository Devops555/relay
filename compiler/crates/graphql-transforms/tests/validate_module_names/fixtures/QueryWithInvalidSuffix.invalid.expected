==================================== INPUT ====================================
#expected-to-throw
query QueryWithInvalidSuffixMutation {
    me {
        name
    }
}
==================================== ERROR ====================================
Queries in graphql tags must start with the module name ('QueryWithInvalidSuffix') and end with 'Query'. Got 'QueryWithInvalidSuffixMutation' instead.:
QueryWithInvalidSuffix.invalid.graphql:2:7:2:37
query QueryWithInvalidSuffixMutation {
